docs/source/references/modules.rst

```diff
@@ -462,6 +462,12 @@ MeanAveragePrecision
 .. autoclass:: torchmetrics.detection.mean_ap.MeanAveragePrecision
     :noindex:
 
+PanopticQuality
```

Comment:
similar instance should be added to `docs/source/references/functional.rst` for the functional counterpart